pkg/api/types.go

```diff
@@ -551,6 +553,20 @@ type RBDVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// DownwardAPIVolumeSource represents a volume containing downward API info
+type DownwardAPIVolumeSource struct {
+	// Items is a list of DownwardAPIVolume file
+	Items []DownwardAPIVolumeFile `json:"items,omitempty"`
+}
+
+// DownwardAPIVolumeFile represents information to create the file containing the pod field
```

Comment:
`DownwardAPIVolumeFile represents a single file containing information from the downward API`